src/copy.js

```diff
@@ -0,0 +1,20 @@
+/* eslint-disable no-console */
+'use strict';
+
+const fs = require('fs');
+
+const copy = (src, dest) => {
+  if (src === dest) {
+    return;
+  }
+
+  try {
+    const fileContent = fs.readFileSync(src, 'utf8');
```

Comment:
Not all files are `utf8`. Also you don't consume file content, so don't specify utf8
```suggestion
    const fileContent = fs.readFileSync(src);
```